src/firefly/html/demo/ffapi-highlevel-test.html

```diff
@@ -23,6 +23,14 @@
     <br>
 </div>
 
+<div>
+    <!--<a href='javascript:firefly.getViewer("testChannel").showImage({url:"http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits"})'>test open</a>-->
+    <a href='javascript:firefly.getViewer().plotURL("http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits")'>test open</a>
```

Comment:
re: 'test open'
Safari:  failed with msg:  Firefly: Operation fail: Could not launch a new browser tab.
Chrome:  blocked by popup blocker
Firefox: works fine.
I know the popup blocker is not a popular complains, but someone will.  If there's an easy way to move the window.open code closer to the clicked event, it should solve that problem.